packages/ui-extensions/src/surfaces/admin/api/customer-segmentation-template/customer-segmentation-template.ts

```diff
@@ -0,0 +1,46 @@
+import type {StandardApi} from '../standard/standard';
+import type {ExtensionPoint as AnyExtensionPoint} from '../../extension-points';
+
+export enum CustomerSegmentationFeature {
```

Comment:
I think there needs to be some documentation about what these are.

Reply:
I can add more context. These are internal feature flags for our own Segmentation templates. Other 3P apps shouldn't rely on them and only use [QL public features](https://help.shopify.com/en/manual/customers/customer-segmentation/customer-segments/customer-segmentation-reference/customer-segment-shopify).

Once we open this to 3Ps, can we expose a new API interface without this property?